test/helpers/lsp.cc

```diff
@@ -508,6 +510,19 @@ vector<unique_ptr<LSPMessage>> getLSPResponsesFor(LSPWrapper &wrapper, unique_pt
     return getLSPResponsesFor(wrapper, move(messages));
 }
 
+namespace {
+
+// Like absl::StripTrailingAsciiWhitespace, but only blank characters (tabs and spaces)
+ABSL_MUST_USE_RESULT inline absl::string_view stripTrailingAsciiBlank(absl::string_view str) {
```

Comment:
Does `[[nodiscard]]` not work here? It looks like abseil suggests using that instead for c++17 and above: https://github.com/abseil/abseil-cpp/blob/0c6302fe427963ec5c471d3ee660120682ab15f7/absl/base/attributes.h#L403-L404

```suggestion
[[nodiscard]] inline absl::string_view stripTrailingAsciiBlank(absl::string_view str) {
```

Reply:
It probably works, I just copied the implementation of `StripTrailingAsciiWhitespace` verbatim with no changes other than the `is_blank` thing.

I'll change this in a follow up so i don't have to wait for tests to build for this PR.